.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -17,7 +17,10 @@ Steps to reproduce the behavior:
 3. Scroll down to '....'
 4. See error
 
-**Expected behavior**
+**Actual outcome**
+A clear and concise description of the impact of the bug.
```

Comment:
```suggestion
A clear and concise description of what actually happened and caused this issue.
```

Reply:
We don't want to know "what caused this issue" in this section - we want to understand the severity. I was considering adding some examples, but in my mind this section is used for descriptors such as:
"The RPC isn't executed and all RPCs thereafter do not work"
"The transport component configuration is lost when entering playmode"

Happy for more suggestions, but I don't think "what caused this issue" is this field.

Reply:
oh, I'm actually after the exact same thing but I guess my wording is wrong.
by "what actually happened", I was trying to ask: "what didn't work?" or "what did break?"

Reply:
It's more than what I expected out of this section. I would have just expected to hear the outcome based on the repro steps.
```
Bug - Multiplayer Room duplication
Description - There seems to be an issue in the lobby system where room IDs aren't being matched. 
Repro Steps - Player A creates Room A as public. Player B searches for Room A. Player B joins Room A.
Actual Outcome - Player B creates a duplicate version of Room A and doesn't see Player A in his room
Expected Outcome - Player B joins Player A in the room.
```
I think the impact would be useful too, I just wouldn't necessarily think of it in the same question